.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+BEFORE POSTING YOUR ISSUE:
+- These comments won't show up when you submit the issue.
+- Try to add as much detail as possible. Be specific!
+- If you're requesting a new feature, explain why you'd like it to be added.
+- Search this repository for issues and pull requests and whether it has been fixed or reported already.
+- Ensure you are using the latest code before logging bugs.
+-->
+
+**Issue Overview**
+This is a brief overview of the issue.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Versions**
+
+* Browser: 
```

Comment:
Do you think the Browser version is needed?

Reply:
Probably not right now but perhaps in the future when there are some UI components as well :-)

Reply:
We could always add it back in then 🤷‍♂️